packages/contracts/contracts/voting/erc20/ERC20VotingSetup.sol

```diff
@@ -203,7 +203,7 @@ contract ERC20VotingSetup is PluginSetup {
             plugin,
             _dao,
             NO_ORACLE,
-            erc20VotingBase.SET_CONFIGURATION_PERMISSION_ID()
+            erc20VotingBase.CHANGE_VOTE_SETTINGS_PERMISSION_ID()
```

Comment:
`CHANGE_PROPOSAL_SETTINGS_PERMISSION_ID`

Reply:
Ah, absolutely. Good catch

Reply:
We tacke this in task https://aragonassociation.atlassian.net/browse/APP-1321